graspologic/match/qap.py

```diff
@@ -455,13 +462,23 @@ def _quadratic_assignment_faq(
 
```

Comment:
It would be be helpful if with each print statement we also get the elapsed time, similar to joblib ([here](https://github.com/joblib/joblib/blob/754433f617793bc950be40cfaa265a32aed11d7d/joblib/parallel.py#L911) for reference). So probably set the start time here.